### ISSUE TITLE
Training Code?

### ISSUE THREAD

**Issue by anurag12-webster** [resemble-ai/chatterbox#45]
This model is awesome looking forward to Training code

**Comment by tzumby** [comment]
+1

**Comment by Aleks-7-7-7** [comment]
+1

**Comment by ankandrew** [comment]
+1

**Comment by TediPapajorgji** [comment]
First off, just want to say a heartfelt thanks for all the love and excitement around this model. We’re genuinely grateful for the open source community, and honestly, a lot of what we do is inspired by your projects, questions, and experiments.

We’ve had quite a few people asking for the training code. I get it, openness is a big part of what makes this space awesome. I want to be real with you: building these models is not cheap. Training, testing, and keeping everything running takes a lot of resources, both time and money. In order to keep this going and actually be able to release more cool stuff in the future, we need a way to keep the project sustainable.

For now, that means we’re not releasing the training code, and fine-tuning will be something we support through our paid API (https://app.resemble.ai). This helps us pay the bills and keep pushing out models that (hopefully) benefit everyone.

We love being part of this community, and we’re doing our best to strike a balance between openness and being able to keep doing the work. We hope you understand. If you have thoughts, feedback, or just want to chat, our door is always open.

**Comment by C00reNUT** [comment]
https://github.com/alisson-anjos/chatterbox-finetune

**Comment by cod3r0k** [comment]
Any update? does is officially well? @C00reNUT

**Comment by C00reNUT** [comment]
https://github.com/stlohrey/chatterbox-finetuning code works fine, I am using it on some of mine private datasets - not sure about the default hyperparameters and the dataset size needed to train the new language... 

There are nice reports about training on dataset in German language - https://wandb.ai/sebastian-bodza/t3-training/reports/Chatterbox-Finetune-600k-data--VmlldzoxMzE1Mzg5MA and https://x.com/SebastianB929/status/1932333058616631522 - however for my dataset i needed to use lower lr rates because even with long warmup phase the training was still unstable with such a high lr rate...

I speak German so I can tell that the quality of the model is good, but I haven't done any massive testing so I am not sure about the stability of the output like word error rates, etc.

**Comment by anjalyv** [comment]
@C00reNUT ,

I am working with Indian languages. The fine-tuned model is generating audio, but the speech does not correspond to the input text. How did you handle tokenization for German?
